src/Apps/Order2/Routes/Checkout/Order2CheckoutRoute.tsx

```diff
@@ -0,0 +1,166 @@
+import ChevronDownIcon from "@artsy/icons/ChevronDownIcon"
+import {
+  Box,
+  Button,
+  Checkbox,
+  Column,
+  Flex,
+  GridColumns,
+  Spacer,
+  Stack,
+  Text,
+} from "@artsy/palette"
+import { useFlag } from "@unleash/proxy-client-react"
+import { addressFormFieldsValidator } from "Components/Address/AddressFormFields"
+import { AddressFormFields } from "Components/Address/AddressFormFields"
+import { ErrorPage } from "Components/ErrorPage"
+import { useSystemContext } from "System/Hooks/useSystemContext"
+import createLogger from "Utils/logger"
+import type { Order2CheckoutRoute_viewer$key } from "__generated__/Order2CheckoutRoute_viewer.graphql"
+import { Formik, type FormikHelpers, type FormikValues } from "formik"
+import type * as React from "react"
+import { useEffect } from "react"
+import { Meta, Title } from "react-head"
+import { graphql, useFragment } from "react-relay"
+import * as yup from "yup"
+
+const logger = createLogger("Order2DetailsRoute.tsx")
+
+interface Order2CheckoutRouteProps {
+  viewer: Order2CheckoutRoute_viewer$key
+}
+export const Order2CheckoutRoute: React.FC<Order2CheckoutRouteProps> = ({
+  viewer,
+}) => {
+  const { isEigen } = useSystemContext()
+
+  const data = useFragment(FRAGMENT, viewer)
+  const isCheckoutRedesignEnabled = useFlag("emerald_checkout-redesign")
+
+  // biome-ignore lint/correctness/useExhaustiveDependencies: placeholder because we aren't using this yet
+  useEffect(() => {
+    logger.warn("Order checkout page data:", {
+      data,
+      isCheckoutRedesignEnabled,
+    })
+  }, [isCheckoutRedesignEnabled])
+
+  if (!isCheckoutRedesignEnabled) return <ErrorPage code={404} />
+
+  const validationSchema = yup.object().shape({
+    ...addressFormFieldsValidator({ withPhoneNumber: true }),
+    saveAddress: yup.boolean(),
+  })
+
+  return (
+    <>
+      <Title>Checkout | Artsy</Title>
+      <Meta
+        name="viewport"
+        content={
+          isEigen
+            ? "width=device-width, user-scalable=no"
+            : "width=device-width, initial-scale=1, maximum-scale=5 viewport-fit=cover"
+        }
+      />
```

Comment:
We shouldn't be messing with the viewport like this on a per-app basis, which overrides default framework behavior.